salt/sshd/sshd_config

```diff
@@ -94,7 +94,7 @@ Subsystem sftp /usr/lib/openssh/sftp-server
 # If you just want the PAM account and session checks to run without
 # PAM authentication, then enable this but set PasswordAuthentication
 # and ChallengeResponseAuthentication to 'no'.
-UsePAM no
+UsePAM yes
```

Comment:
Is this really needed for this patch? Why?

Reply:
This is needed to have the user change it password upon the first login.